### DISCUSSION TITLE
CSS alternative

### DISCUSSION THREAD

**Discussion by drxcheng** [xyflow/xyflow#4206] [Q&A] [answered]
our project is using jss instead of css and there is no config for css in webpack.

I understand that `'reactflow/dist/style.css'` is needed, but is there any other alternative?

**Comment by moklick** [comment]
you can add a css loader to webpack or convert the style to jss (https://github.com/xyflow/xyflow/blob/v11/packages/core/src/styles/init.css + https://github.com/xyflow/xyflow/blob/v11/packages/core/src/styles/style.css).

**Reply by drxcheng** [reply]
Thanks. I'll try css loader approach. I was wondering if there is any native solution.

**Reply by drxcheng** [reply, marked as answer]
adding style-load and css-loader to Webpack works
